src/wp-admin/includes/upgrade.php

```diff
@@ -654,6 +654,8 @@ function wp_upgrade() {
 			update_site_meta( get_current_blog_id(), 'db_last_updated', microtime() );
 		}
 
+		delete_transient( 'wp_core_block_styles' );
```

Comment:
```suggestion
		delete_transient( 'wp_core_block_css_files' );
```

Reply:
Also, per below, wouldn't site transients be more appropriate?

```suggestion
		delete_site_transient( 'wp_core_block_css_files' );
```

Reply:
@westonruter Using network options ( tranisents ) has a performance impact. I think network options are not autoloaded. So it result in a database query, slow things down. Maybe we could work this out later.